spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/AxonAutoConfiguration.java

```diff
@@ -165,11 +165,10 @@ private Serializer buildSerializer(RevisionResolver revisionResolver,
             case XSTREAM:
             case DEFAULT:
             default:
-                XStreamSerializer xStreamSerializer = XStreamSerializer.builder()
-                                                                       .revisionResolver(revisionResolver)
-                                                                       .build();
-                xStreamSerializer.getXStream().setClassLoader(beanClassLoader);
-                return xStreamSerializer;
+                return XStreamSerializer.builder()
```

Comment:
Shouldn't this be done for the `JacksonSerializer` too?

Reply:
This might lead to override class loader in Spring Boot auto configured `ObjectMapper` (configured in `JacksonAutoConfiguration`). I think it's not good.

Reply:
Fair point. That makes it something we simply cannot do in the auto configuration on the `JacksonSerializer` directly.

We could take control of the `defaultAxonObjectMapper` bean, as that's already being searched for any how, but that's not really the scope of this issue.